INSTRUCTIONS.md

```diff
@@ -0,0 +1,97 @@
+# Using Anaconda
+
+IMPACT-T is available through conda-forge and can be installed via:
+```bash
+conda create -n impact
+source activate impact # or conda activate impact
+# For non-MPI
+conda install -c conda-forge impact-t
+
+# For OpenMPI
+conda install -c conda-forge impact-t=*=mpi_openmpi*
+
+# For MPICH
+conda install -c conda-forge impact-t=*=mpi_mpich*
+```
+# Compiling The Code
+
```

Comment:
I use these short lines everywhere to get totally fresh CMake users in no-time up-to-speed:
```suggestion

If you are new to CMake, [this short tutorial](https://hsf-training.github.io/hsf-training-cmake-webpage/) from the HEP Software foundation is the perfect place to get started with it.

If you just want to use CMake to build the project, jump into sections *1. Introduction*, *2. Building with CMake* and *9. Finding Packages*.

```